.gitignore

```diff
@@ -23,4 +23,5 @@ testbin/*
 *.swp
 *.swo
 *~
+.vscode/*
```

Comment:
I think it was pushed by mistake, need to be removed.

Reply:
Ok shall remove it. Thought I would keep it as it would avoid all vscode files in general.

Reply:
Okay. If it's avoided you can keep it. No issue then.